src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -1401,6 +1401,9 @@ bool PerfMemoryLinux::restore() {
 
   int vmid = os::current_process_id();
   char* user_name = get_user_name(geteuid());
+  if (!user_name) {
+    return false;
```

Comment:
Could we accompany this with an error message?

Reply:
I'm not sure about an error message as it doesn't fail restore procedure. 
The same scenario from [here ](https://github.com/openjdk/crac/blob/crac/src/hotspot/os/posix/perfMemory_posix.cpp#L1032) doesn't show any message also.

Reply:
OK, that scenario would end up with an error message only with [develop flags](https://github.com/openjdk/crac/blob/crac/src/hotspot/os/posix/perfMemory_posix.cpp#L1261).